pam/pam_adsys.c

```diff
@@ -25,6 +25,7 @@
 
 #define _GNU_SOURCE
 
+#include <ctype.h>
```

Comment:
We use `toLower()` to normalize the username, but we weren't including the `ctype.h` header, which ended up in a implict declaration (oops)